### PR TITLE
Implement spellchecking on Linux

### PR DESCRIPTION
**Pull request by mymedia2**
See also: https://github.com/desktop-app/lib_spellcheck/pull/1

### CODE REVIEW

File: Telegram/SourceFiles/settings/settings_advanced.cpp [hunk 1]
```diff
@@ -32,6 +32,10 @@ For license and copyright information please follow this link:
 #include "app.h"
 #include "styles/style_settings.h"
 
+#ifndef TDESKTOP_DISABLE_SPELLCHECK
+# include "spellcheck/platform/platform_spellcheck.h"
```
Comment (23rd) [inline Telegram/SourceFiles/settings/settings_advanced.cpp]: There is an extra space at the beginning of the line.

**Comment by kirsan31** [comment]
@23rd 
https://github.com/telegramdesktop/tdesktop/issues/96#issuecomment-550201078
Any ETA on Win7 support?

**Comment by Aokromes** [comment]
windows 7 enters EOL next year, there is no point of care to much about it.

**Comment by 23rd** [comment]
@kirsan31 
I believe it will be done in December.

@Aokromes 
I disagree. It makes sense to support Windows 7 as long as Firefox and Chrome do so.

File: Telegram/SourceFiles/settings/settings_advanced.cpp (hunk 1 above)
Comment (mymedia2) [reply Telegram/SourceFiles/settings/settings_advanced.cpp]: It was an indentation pointing to `if` block. But I've just removed it.

**Comment by 23rd** [comment]
@mymedia2 Please, remove the update of the `lib_spellcheck` submodule, this will be updated with another commit.

**Comment by 23rd** [comment]
@kirsan31 I'm sorry, not this year. =(

**Comment by kirsan31** [comment]
> @kirsan31 I'm sorry, not this year. =(

@23rd No problem - happy N.Y. :)

**Comment by john-preston** [comment]
@mymedia2 I've rebased-merged it in command line, thanks!

**Pull request closed without merging**